crates/util/src/paths.rs

```diff
@@ -12,47 +12,27 @@ lazy_static::lazy_static! {
     pub static ref CONVERSATIONS_DIR: PathBuf = CONFIG_DIR.join("conversations");
     pub static ref EMBEDDINGS_DIR: PathBuf = CONFIG_DIR.join("embeddings");
     pub static ref THEMES_DIR: PathBuf = CONFIG_DIR.join("themes");
-    pub static ref LOGS_DIR: PathBuf = if cfg!(target_os="macos") {
+    pub static ref LOGS_DIR: PathBuf = if cfg!(target_os = "macos") {
         HOME.join("Library/Logs/Zed")
     } else {
         CONFIG_DIR.join("logs")
     };
-    pub static ref SUPPORT_DIR: PathBuf = if cfg!(target_os="macos") {
+    pub static ref SUPPORT_DIR: PathBuf = if cfg!(target_os = "macos") {
         HOME.join("Library/Application Support/Zed")
     } else {
-        CONFIG_DIR.join("support")
+        CONFIG_DIR.clone()
```

Comment:
This was an intentional change.

It felt weird to have a `support` directory inside of `~/.config/zed`, so we can just flatten all of the directories right inside of it.